x/launch/client/cli/tx_settle_request.go

```diff
@@ -0,0 +1,108 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdSettleRequest() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "settle-request [chainID] [requestID] [approve]",
```

Comment:
I think it can be dangerous to use a boolean value for `approve` because I may think `ParseBool` can give the wrong value from a simple typing mistake

What I would do is to have, as the first positional argument, a string that can be either `approve` or `reject`
```
settle-request [approve|reject] [chainID] [requestID]
```

Another solution can be to have two commands `approve-request` and `reject-request` but I think the solution above is sufficient